### DISCUSSION TITLE
Set description and amount when generating invoice with Google sheets

### DISCUSSION THREAD

**Discussion by virginiavannini** [fattureincloud/api#92] [Q&A] [answered]
Hi, me again.

I'm using Google sheet to generate Invoices.
All is ok, the invoice is generated, but without description, vat summary and all the Amount (all the bottom part).

This is my function, as you can see I've the amountNet but I can't see it in the invoice.

// This function makes the actual API call to create the invoice
function createInvoice(clientName, clientVatNumber, taxCode, addressStreet, addressPostalCode, addressCity, addressProvince, addressExtra, country, certifedEmail, eiCode, type, amountNet, amountGross, subject, visibleSubject, dateInvoice) {
    var url = "https://api-v2.fattureincloud.it";
    var endpoint = "/c/12345/issued_documents";
    var headers = {
        "Content-Type": "application/json",
        "Authorization": "Bearer ID"
    };

    var body = {
        data: {
            type: "invoice",
            subject : subject,
            visible_subject: visibleSubject,
            amount_net: amountNet,
            amount_vat: 0,
            amount_gross: amountGross,
            entity: {
                name: clientName,
                vat_number: clientVatNumber,
                tax_code : taxCode,
                address_street: addressStreet,
                address_postal_code: addressPostalCode,
                address_city: addressCity,
                address_province: addressProvince ,
                address_extra: addressExtra,
                country: country,
                certified_email: certifedEmail,
                ei_code: eiCode,
                type: type
            },
            date : dateInvoice
        }
    }
    var options = {
        "method": "post",
        "headers": headers,
        "payload": JSON.stringify(body)
    };

    try {
        var response = UrlFetchApp.fetch(url + endpoint, options);
        data = JSON.parse(response.getContentText());
        SpreadsheetApp.getUi().alert("Invoice created succesfully with id: " + data.data.id)

    } catch (e) {
        SpreadsheetApp.getUi().alert(e.message)
    }
}

I've also tested the example (step two: https://developers.fattureincloud.it/docs/quickstarts/apps-script-quickstart ) and in this case I can't see any amount in the invoice. Any suggestion?

**Comment by valmoz** [comment, marked as answer]
Hi @virginiavannini 
have you seen the [Invoice reation Guide](https://developers.fattureincloud.it/docs/guides/invoice-creation)?

Thanks
Mauro

**Reply by valmoz** [reply]
Hi @virginiavannini were you able to solve this issue?

**Reply by valmoz** [reply]
Hi @virginiavannini I'm setting this question as answered due to inactivity.
If you weren't able to solve your issue please feel free to contact us back.

Thanks
Mauro
